json-rpc/types/src/views.rs

```diff
@@ -140,6 +140,7 @@ pub struct AccountView {
     pub delegated_withdrawal_capability: bool,
     pub is_frozen: bool,
     pub role: AccountRoleView,
+    pub version: Option<u64>,
```

Comment:
Do you want to add an annotation to this to not render it if its None?

Reply:
Added. It's nice to have. As server side constructor is not optional, and will always render.